src/chigame/users/views.py

```diff
@@ -72,7 +74,23 @@ def user_history(request, pk):
     try:
         user = User.objects.get(pk=pk)
 
-        return render(request, "users/user_history.html", {"user": user})
+        match_count = Match.objects.filter(players__in=[user]).count()
+        match_wins = Player.objects.filter(Q(user=user, outcome=Player.WIN) | Q(team=user, outcome=Player.WIN)).count()
+
+        tournament_count = Tournament.objects.filter(players__in=[user]).count()
```

Comment:
Hm this isn't quite right because it's just counting if the player is currently in a tournament (could be on going). If you could add another filter where the tournament_end_date < timezone.now() that should make this work.

Reply:
That's a good filter I wasn't aware of, I'll use that